runtime/compiler/env/VMJ9Server.cpp

```diff
@@ -1414,6 +1414,12 @@ TR_J9ServerVM::needsInvokeExactJ2IThunk(TR::Node *callNode, TR::Compilation *com
       }
    }
 
+TR_ResolvedMethod *
+TR_J9ServerVM::createMethodHandleArchetypeSpecimen(TR_Memory *trMemory, TR_OpaqueClassBlock *archetype, uintptrj_t *methodHandleLocation, TR_ResolvedMethod *owningMethod)
+   {
```

Comment:
This will work for now because this version is used only during the creation of the compilee and we don't save any messages anyway, a more generic version could be written (using the J9VMBase code as a template) that could save a little bit of time at the client.
For the compilee I would argue that we could even save the message (not to be done in this PR) by sending the required information in the methodDetails.